tsconfig.json

```diff
@@ -7,10 +7,11 @@
     "target": "ES2018",
     "module": "commonJS",
     "moduleResolution": "node",
+    "esModuleInterop": true,
     "lib": ["ES2018"],
     // We don't make use of tslib helpers, all syntax used is supported by target engine
     "importHelpers": false,
-    "noEmitHelpers": true,
+    "noEmitHelpers": false,
```

Comment:
were you intending to commit the changes to this file?

Reply:
Fixed! thanks